src/dashboard/src/components/administration/urls.py

```diff
@@ -28,6 +28,8 @@
     (r'dips/contentdm/$', 'contentdm_dips'),
     (r'sources/$', 'sources'),
     (r'storage/$', 'storage'),
+    (r'usage/$', 'usage'),
+    (r'usage/clear/(?P<dir_id>\w+)/$', 'usage_clear'),
```

Comment:
Maybe it should be available both in the UI and as an API? It's in the UI so folks using AM in the cloud can do some basic housekeeping without SSH access, but it definitely would make sense to have accessible via the API for automation.